bisq-easy/src/main/java/bisq/bisq_easy/BisqEasyNotificationsService.java

```diff
@@ -144,11 +182,61 @@ public void dismissNotification() {
         updateNotificationVisibilityState();
     }
 
+    private void handleNotification(@Nullable ChatNotification notification) {
+        if (notification == null) {
+            return;
+        }
+        scheduleRefresh();
+    }
+
+    private void scheduleRefresh() {
```

Comment:
I don't see the need for that scheduler. The trade and trade channels will not trigger lots of updates, nor the notifications themself.